docker-run/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2020 Abdud Dayan Adeeb
```

Comment:
If the action code got changed after the forking we should add our copyright here too
```suggestion
Copyright (c) 2020 Abdud Dayan Adeeb
Copyright (c) 2021 Greenbone Networks GmbH
```